ServerRecording/README.MD

```diff
@@ -6,83 +6,96 @@ products:
+- An Azure account with an active subscription. For details, see here to [create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- [Create container registry](https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/deploy-spring-boot-java-app-on-linux#create-an-azure-container-registry-to-use-as-a-private-docker-registry)
+- [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource\'s **connection string** for this quickstart.
+- [Create a webhook](https://docs.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/download-recording-file-sample) and subscribe to the recording events.
 
 ## Code structure
-
-- ./ServerRecording/src/main/java/com/acsrecording/api/controllers : Server app core logics to make Api calls that connects with the Azure Communication Services Web Calling SDK
-- ./ServerRecording/pom.xml : XML file which contains project and package configurations
-- ./ServerRecording/src/main/resources/config.properties : config file which contains user level configurations
+- ./src/main/java/com/acsrecording/api/Controller : Server app core logic to make Api calls that connect with the Azure Communication Services Web Calling SDK
+- ./pom.xml : XML file which contains project and package configurations
+- ./src/main/resources/config.properties : config file which contains user level configurations
 
 ## Before running the sample for the first time
 1. Get the `Connection String` from the Azure portal. For more information on connection strings, see [Create an Azure Communication Resources](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource)
-2. Once you get the config keys add the keys to the **resources/config.properties** file found under the ./ServerRecording/src/main/resources folder.
-	- Input your connection string in the variable: `Connectionstring`
-	- Input your blob storage connectionString string in the variable: `BlobStorageConnectionString`
+2. Once you get the config keys, add the keys to the **resources/config.properties** file found under the ./src/main/resources folder.
+	- Input your ACS connection string in the variable: `ACSConnectionString`
+	- Input your blob storage connection string in the variable: `BlobStorageConnectionString`
 	- Input blob container name for recorded media in the variable `ContainerName`
 	- Input recording callback url for start recording api in the variable `CallbackUri`
 
 ## Locally deploying the sample app
 
-1. Build java code : mvn clean install  
+1. Build java code : `mvn clean install`
 
-2. Run app locally : mvn spring-boot:run
+2. Run app locally : `mvn spring-boot:run`
 
-3. Use postman or any debugging tool and open url - http://localhost:8080 
+3. Use [Postman](https://www.postman.com/) or any debugging tool and open url - http://localhost:8080 
```

Comment:
@ravithanneeru - I just went through this as well. 
I think that advising to 'open url - http://localhost:8080' is insufficient instructions. 
Firstly, that just returns a 404 as there is no default route.
Secondly, all of the controller endpoints require a parameter - e.g. serverCallId in .....

`@GetMapping("/startRecording")`
`public StartCallRecordingResult startRecording(String serverCallId)`

I tried a few different ways to reference serverCallId, but since it is not directly stated here, I wasn't sure how. When I went to reference Spring Boot, it seems that serverCallId would need to be specified in the GetMapping via something like `@GetMapping("/startRecording/{serverCallId}")`

I fixed similar things to what I suggested and what was fixed in the C# quickstart, but can you please continue this based on my comments above?

cc @bharat-kalyan-namburi